svc-telemetry/tcommon/common.go

```diff
@@ -137,15 +138,18 @@ func ContactPlugin(req PluginContactRequest, errorMessage string) ([]byte, strin
 	var resp ResponseStatus
 	var response *http.Response
 	var err error
+	log.Info("INSIDE contact client")
```

Comment:
Please remove debug logs in this file, if not required.

Reply:
addressed